src/jupyter_contrib_nbextensions/nbextensions/exercise/readme.md

```diff
@@ -24,7 +24,7 @@ The extensions provide
 - a cell widget -- A plus/minus button in `exercise` and a sliding checkbox in `exercise2`.
 
 The menubar button is devoted to the creation or removing of the solution. The solution consists in several consecutive cells that can be selected either by multicell selection (*Shift-J* (select next) or *Shift-K* (select previous) keyboard shortcuts --
-*Shift-up* and *Shift-down* will probably work in a near future) or using the [rubberband extension](https://github.com/ipython-contrib/jupyter_contrib_nbextensions/wiki/Rubberband).
+*Shift-up* and *Shift-down* will probably work in a near future) or using the rubberband extension.
```

Comment:
Shift-up/-down works now, so this should probably be the default. Who can remember shift-j/-k ?

Reply:
good point :+1: